made security changes to script in detect-remove-malware-virustotal.rst (#8697)
wazuh/wazuh-documentation: source/proof-of-concept-guide/detect-remove-malware-virustotal.rst

#. Create an active response script ``remove-threat.py`` to remove a file from the Windows endpoint:

   .. warning::

>       This script is intended as a proof of concept (PoC) only. Review and validate the script to ensure it meets the operational and security needs of your environment.

Reviewer: ```suggestion

      This script is a proof of concept (PoC). Review and validate it to ensure it meets the operational and security requirements of your environment.
```
